### PR TITLE
Protractor

### PR DESCRIPTION
**Pull request by lamara**
PR ready.

Depends upon:

- [ ] https://github.com/digitalbazaar/bedrock-protractor/pull/6

### CODE REVIEW

**Comment by mattcollier** [comment]
@lamara merge conflicts here.  Please do `git rebase origin/master`

**Comment by lamara** [comment]
Rebased.

File: lib/services.identity.js [hunk 1]
```diff
@@ -360,8 +360,8 @@ function addRoutes(app) {
             return callback(err);
           }
           data.vars = vars;
-          vars.identity = data.identity;
-          vars.identityMeta = data.identityMeta;
+          //vars.identity = data.identity;
```
Comment (mattcollier) [inline lib/services.identity.js]: remove?

File: test/bower.json [hunk 2]
```diff
@@ -42,4 +42,4 @@
     "lodash": "^4.0.0",
     "mocha": "^2.5.3"
   }
-}
+}
```
Comment (mattcollier) [inline test/bower.json]: need another line here.

File: test/package.json [hunk 3]
```diff
@@ -68,4 +68,4 @@
     "uuid": "^2.0.0",
     "supertest": "^1.0.1"
   }
-}
+}
```
Comment (mattcollier) [inline test/package.json]: need another line here.

**Comment by mattcollier** [comment]
This was merged into: https://github.com/digitalbazaar/bedrock-idp/pull/45

**Pull request closed without merging**